docs/content/documentation/getting-started/configuration.md

```diff
@@ -32,7 +31,19 @@ highlight_code = false
 
 # The theme to use for code highlighting.
 # See below for list of allowed values.
+# This will put the colors of the theme directly in your html.
 highlight_theme = "base16-ocean-dark"
+# For using CSS class definitions in the higlighting, you can use the
+# special theme "css". This is especially useful for dark/light themes,
+# or letting the user decide the higlighting scheme.
+
+# If the site uses some of the predefined syntax highlighing schemes as
+# CSS, you can let Zola generate the CSS.
+generate_theme_css = [
```

Comment:
Hmmm.
I think the inner part is ok, except maybe file -> filename for clarity.

As for `generate_theme_css`, I think `highlighting_themes_css` is more consistent.

Reply:
Yepp, much better